test_runner/src/main/kotlin/ftl/reports/xml/model/JUnitTestSuite.kt

```diff
@@ -29,11 +32,11 @@ data class JUnitTestSuite(
     val timestamp: String?, // String. Android only
 
     @JacksonXmlProperty(isAttribute = true)
-    val hostname: String, // String.
+    val hostname: String? = null, // String.
```

Comment:
Just for my understanding, are we ok with possible `null` here? Previous implementation did not predicted such value. Thanks!

Reply:
I have asked @bootstraponline about that, he said https://github.com/Flank/flank/issues/545#issuecomment-597885474. So I decide to change it to nullable here, because I didn't find hostname in API

Reply:
Got it, thanks for explanation